src/multi_vector_simulator/C1_verification.py

```diff
@@ -766,3 +770,72 @@ def check_for_sufficient_assets_on_busses(dict_values):
+
+    Notes
+    -----
+    Tested with:
+    - test_check_energy_system_can_fulfill_max_demand_sufficient_capacities()
+    - test_check_energy_system_can_fulfill_max_demand_no_maximum_capacity()
+    - test_check_energy_system_can_fulfill_max_demand_insufficient_capacities()
+    - test_check_energy_system_can_fulfill_max_demand_with_storage()
+    """
+    for bus in dict_values[ENERGY_BUSSES]:
+        skip_check = False
+        opt_cap_storage = False
+        peak_demand = 0
+        peak_generation = 0
+        for item in dict_values[ENERGY_BUSSES][bus][ASSET_DICT]:
+            # filters out excess energy sinks, leaving only actual demand profiles
+            if item in dict_values[ENERGY_CONSUMPTION] and \
+                    dict_values[ENERGY_CONSUMPTION][item][DISPATCHABILITY][VALUE] is False:
+                peak_demand += dict_values[ENERGY_CONSUMPTION][item][TIMESERIES_PEAK][VALUE]
+            for group in [ENERGY_CONVERSION, ENERGY_PRODUCTION]:
+                if item in dict_values[group]:
+                    peak_generation += dict_values[group][item][INSTALLED_CAP][VALUE]
+                    if dict_values[group][item][OPTIMIZE_CAP][VALUE] is True:
+                        if dict_values[group][item][MAXIMUM_CAP][VALUE] is None:
+                            skip_check = True
+                        else:
+                            peak_generation += dict_values[group][item][MAXIMUM_CAP][VALUE]
+            if item in dict_values[ENERGY_STORAGE]:
+                peak_generation += dict_values[ENERGY_STORAGE][item][OUTPUT_POWER][INSTALLED_CAP][VALUE]
+                if dict_values[ENERGY_STORAGE][item][OPTIMIZE_CAP][VALUE] is True:
+                    # unlike the conversion/production assets, no maximum capacities are defined for
+                    # storage assets and therefore as soon as a storage asset is connected, the check
+                    # should pass
+                    skip_check = True
+                    opt_cap_storage = True
+# return peak_generation and peak_demand as values
+        if peak_generation < peak_demand and skip_check is False:
+            logging.warning(f"The assets of {bus} might have insufficient capacities to fulfill"
+                            f" the maximum demand ({round(peak_generation)}<{round(peak_demand)}).")
+        elif opt_cap_storage is True:
+            logging.debug(f"The check for assets having sufficient capacities to fulfill the"
+                          f" maximum demand has successfully passed for bus {bus}. However,"
+                          f" this check does not determine if the storage can be sufficiently"
+                          f" charged by the other assets in the energy system.")
+        else:
+            logging.debug(f"The check for assets having sufficient capacities to fulfill the"
+                          f" maximum demand has successfully passed for bus {bus}.")
```

Comment:
```suggestion
            logging.debug(f"The check for assets having sufficient capacities to fulfill the"
                          f" maximum demand has successfully passed for bus {bus}. At least partially, this also happens because there is a storage asset that is being optimized."
                          f" However, this check does not determine if the storage can be sufficiently"
                          f" charged by the other assets in the energy system to ensure a viable supply for each timestep.")
        else:
            logging.debug(f"The check for assets having sufficient capacities to fulfill the"
                          f" maximum demand has successfully passed for bus {bus}.")
```

Reply:
Add an `else` here with `logging.debug()` if the test passes (makes it explicit)